pinkopy/base_session.py

```diff
@@ -137,7 +137,7 @@ def request(self, method, path, attempt=None, headers=None, payload=None,
             else:
                 log.info('CMDBSession made request to {0}'.format(url))
                 return res
-        except requests.exceptions.HTTPError as err:
+        except requests.HTTPError as err:
```

Comment:
I got an AttributeError when using `requests.exceptions.HTTPError`. I agree that it would be more readable.